Lekce_02_Ukol04/Lucistnik.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Lekce_02_Ukol04
+{
+    internal class Lucistnik
+    {
+        private int PocetSipu;
+
+        public Lucistnik(int pocetSipu)
+        {
+            if (pocetSipu >= 0)
+            {
+                PocetSipu = pocetSipu;
+            }
+            else
+            {
+                Console.WriteLine("Počet šípů nemůže být negativní.");
+            }
+        }
+
+        public void Vystrel()
+        {
+            PocetSipu--;
+            if (PocetSipu > 0)
+            {
+                Console.WriteLine("Vždy se trefím přímo doprostřed!");
+            }
+            else 
+            {
+                Console.WriteLine("Nemám šípy.");
+            }
+        }
+
+        public int NastavPocetSipu()
```

Comment:
nit: Tahle metoda nenastavuje pocet sipu (nijak ho nemeni), pouze vraci pocet sipu. Takze by se to mohlo jmenovat VratPocetSipu() :)

Reply:
opraveno